bin/multielasticdump

```diff
@@ -337,6 +337,7 @@ if (options.direction === 'load') {
       .replace(`.settings.${fileExt}`, '')
       .replace(`.template.${fileExt}`, '')
       .replace(`.${fileExt}`, ''))
+      .replace('/', '')
```

Comment:
@ferronrsmith I think you meant to do this instead:
```
      .replace(`.${fileExt}`, '')
      .replace('/', ''))
```

Reply:
Right